app/src/main/java/com/evanfuhr/pokemondatabase/activities/MainActivity.java

```diff
@@ -211,6 +210,16 @@ public void onNothingSelected(AdapterView<?> parent) {
                     public void onClick(DialogInterface dialog, int id) {
                         // Save game version
                         dialog.dismiss();
+
+                        // We need an Editor object to make preference changes.
+                        // All objects are from android.context.Context
+                        SharedPreferences settings = getSharedPreferences(String.valueOf(R.string.gameVersionID), MODE_PRIVATE);
```

Comment:
Seems kind of strange to stringify something that says string already in it `R.string.gameVersionID`. Any way to combine `String.valueOf()` with `R.string.gameVersionID`?

Reply:
I think the object getting returned isn't itself a String, I'll take a look for refactoring